linalg/hypre.cpp

```diff
@@ -967,17 +974,26 @@ static void MakeWrapper(const hypre_CSRMatrix *mat, SparseMatrix &wrapper)
    wrapper.Swap(tmp);
 }
 
+
 void HypreParMatrix::GetDiag(SparseMatrix &diag) const
 {
    MakeWrapper(A->diag, diag);
 }
 
+
 void HypreParMatrix::GetOffd(SparseMatrix &offd, HYPRE_Int* &cmap) const
 {
    MakeWrapper(A->offd, offd);
    cmap = A->col_map_offd;
 }
 
+
+void HypreParMatrix::GetProcRows(SparseMatrix &colCSRMat)
```

Comment:
This function does not appear to be used anywhere in this PR. Is it generally useful enough to be included?

Reply:
I generally think better access to hypre functionality makes linear algebra and solver work in MFEM easier. This particular function would not come up often for a regular user, but I added it for solver development and testing.

Reply:
Why is it called `colCSRMat`?

Reply:
I don't have a strong preference here, but it is unusual in mfem to have an unused function. @v-dobrev do you think this should be kept? If it is kept, maybe a simpler name for `colCSRMat` should be used.

Reply:
I think it is okay to provide convenience wrappers like this for hypre functions.

However, in cases like this I think it is better to use the hypre original name and call this method `MergeDiagAndOffd` and respectively use a parameter name like `merged` instead of `colCSRMat`.